source/configuration_manual/mail_location/index.rst

```diff
@@ -83,6 +83,12 @@ Key                Value Description
                      The mapping between mailbox names and GUIDs exists in
                      ``dovecot.list.index*`` files.
 
+``KEEP-NOSELECT``  Do not automatically delete ``\NoSelect`` mailboxes that have
+                   no children. This is the inverse of ``NO-NOSELECT``, which
+                   is the now the default,.
+
```

Comment:
"is the now the default" -> "was previously the default".

Reply:
mmmmh the default is NO-NOSELECT. KEEP-NOSELECT represents what was previously the default.
I guess my wording was confusing, so I remove the reference to the default in KEEP-NOSELECT and keep it only in NO-NOSELECT which is hte new default.